.github/workflows/ci.yml

```diff
@@ -24,36 +24,42 @@ jobs:
   build-images:
     strategy:
       matrix:
-        docker-platform: ["linux/arm64", "linux/amd64"]
-        images: ["rocky9.2-10-gpu", "rocky9.2-10-slim", "rocky9.2-10-cpu"]
+        architecture: ["arm64", "amd64"]
+        image: ["wolfi-base", "rocky9.2-10-gpu", "rocky9.2-10-slim", "rocky9.2-10-cpu"]
+        exclude:
+          - architecture: arm64
+            image: wolfi-base
     runs-on: ubuntu-latest-m
     needs: [set-short-sha]
     env:
       SHORT_SHA: ${{ needs.set-short-sha.outputs.short_sha }}
     steps:
       - uses: actions/checkout@v3
-      - name: Login to Quay.io
-        uses: docker/login-action@v2
-        with:
-          registry: quay.io
-          username: ${{ secrets.QUAY_IO_ROBOT_USERNAME }}
-          password: ${{ secrets.QUAY_IO_ROBOT_TOKEN }}
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v2
+      - name: Download APKs for chainguard/wolfi-base
+        if: matrix.architecture == 'amd64' && matrix.image == 'wolfi-base'
+        run: make docker-dl-packages
       - name: Build base images
-        run: |
-          make build-base-images
+        run: make build-base-images
         env:
-          DOCKER_PLATFORM: ${{ matrix.docker-platform }}
-          DOCKERFILE: ${{ matrix.images }}
+          DOCKER_PLATFORM: linux/${{ matrix.architecture }}
+          DOCKERFILE: ${{ matrix.image }}
           CI: "true"
       - name: Print images
         run: docker images
+      - name: Scan image for vulnerabilities
+        uses: anchore/scan-action@v3
+        with:
+          image: "${{ env.DOCKER_BUILD_REPOSITORY }}:${{ matrix.image }}-${{ matrix.architecture }}-${{ env.SHORT_SHA }}"
+          severity-cutoff: high
+          fail-build: ${{ matrix.image == 'wolfi-base' }}
```

Comment:
rocky linux images have numerous vulnerabilities but I assume we don't want to fail PRs because of that. WDYT?

Reply:
Yeah I think we're good to just apply this to wolfi